core/src/main/java/io/parsingdata/metal/token/Seq.java

```diff
@@ -27,6 +27,13 @@
 import io.parsingdata.metal.data.ParseResult;
 import io.parsingdata.metal.encoding.Encoding;
 
+/**
+ * A {@link Token} that specifies a dependency between a list of tokens.
```

Comment:
Agreed but it's a bit theoretical. I mean sure it's technically an AND, but most users won't care I think...